src/app/page.jsx

```diff
@@ -33,6 +39,12 @@ export default function Home() {
 				) : (
 					<LoginForm onToggle={() => toggleForgotPassword(true)} />
 				)}
+				<button
+					onClick={handleDonationClick}
+					className="mt-4 bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded flex flex-col items-center"
+				>
+					Donar €
+				</button>
```

Comment:
This button is hard to see
![imagen](https://github.com/ISPP-07/frontend-acat/assets/73228905/3dbb04ec-d050-49e4-a0f9-e1ad41177988)

I would suggest putting it below the ACAT image and making it a bit nicer and flashy, with a "Donate to ACAT to help needed people get their life back" message along with it or something of the sort